File: patterns/javascript-patterns/pattern48.js
// Pattern 48: ...

//     1
//    **
//   333
//  ****
// 55555

const n = 5;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = n; j >= 1; j--) {
        if (i >= j) {
            if (i % 2 === 0) {
                row += '*';
            } else {
                row += i;
            }
        } else {
            row += ' ';
        }
    }

    console.log(row);
}

File: patterns/python-patterns/pattern48.py
# Pattern 48: ...

#     1
#    **
#   333
#  ****
# 55555

n = 5

for i in range(1, n + 1):
    for j in range(n, 0, -1):
        if i >= j:
            if i % 2 == 0:
                print("*", end="")
            else:
                print(i, end="")
        else:
            print(" ", end="")
    print()